released/packages/coq-rupicola/coq-rupicola.0.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+authors: [
+  "Massachusetts Institute of Technology"
+]
+maintainer: "Jason Gross <jgross@mit.edu>"
+homepage: "https://github.com/mit-plv/rupicola"
+bug-reports: "https://github.com/mit-plv/rupicola/issues"
+license: "MIT"
+build: [
+  [make "-j%{jobs}%" "EXTERNAL_DEPENDENCIES=1" "all" FIXME EXTERNAL_DEPS]
+]
+install: [make "EXTERNAL_DEPENDENCIES=1" "install" FIXME EXTERNAL_DEPS]
+depends: [
+  "coq" {>= "8.15~"}
```

Comment:
Isn't that implied by coq-bedrock2?

Reply:
Will Rupicola always have the same Coq version compatibility as the corresponding version of bedrock2?  I figure it's better to specify explicitly

Reply:
Indeed we prefer if some Coq version bound is explicitly specified for each pacakge.